build.gradle

```diff
@@ -151,6 +151,8 @@ dependencyManagement {
 }
 
 dependencies {
+  compile group: 'com.microsoft.azure', name: 'azure-storage', version: '4.2.0'
```

Comment:
Any reason we are not using 
`compile group: 'com.microsoft.azure', name: 'azure-storage', version: '7.0.0'
`